libs/react-helpers/src/hooks/use-theme-switcher.ts

```diff
@@ -1,42 +1,77 @@
-import { useEffect, useState } from 'react';
+import create from 'zustand';
 import { LocalStorage } from '../lib/storage';
 
-const darkTheme = 'dark';
-const lightTheme = 'light';
-type themeVariant = typeof darkTheme | typeof lightTheme;
+const THEME_STORAGE_KEY = 'theme';
+const Themes = {
+  DARK: 'dark',
+  LIGHT: 'light',
+} as const;
 
-const darkThemeCssClass = darkTheme;
+type Theme = typeof Themes[keyof typeof Themes];
 
-const getCurrentTheme = () => {
-  const theme = LocalStorage.getItem('theme');
-  if (
-    theme === darkTheme ||
-    (!theme &&
-      typeof window !== 'undefined' &&
-      window.matchMedia('(prefers-color-scheme: dark)').matches)
-  ) {
-    return darkTheme;
+const validateTheme = (theme: string): theme is Theme => {
+  if (theme === 'light' || theme === 'dark') return true;
```

Comment:
```suggestion
  if ([Themes.LIGHT, Themes.DARK].includes(theme)) return true;
```

Reply:
Or like that...

Reply:
or one better `Object.values(Themes).includes(theme)`